### DISCUSSION TITLE
Java heap Space Error

### DISCUSSION THREAD

**Discussion by Boss4644** [nextgenhealthcare/connect#4982] [Q&A]
unable to read CSV file with more then 1MB size, its showing the below error

Transformer error
ERROR MESSAGE: Error evaluating transformer
java.lang.OutOfMemoryError: Java heap space

**Comment by jonbartels** [comment]
https://forums.mirthproject.io/articles/177082-forum-etiquette

> What steps have you taken? What issue are you having? i.e. what's not working? Have code that's not working? Share it here.
